src/edgepi/calibration/edgepi_eeprom.py

```diff
@@ -192,16 +195,15 @@ def __parameter_sanity_check(self, mem_addr: int = None,
             length (int): length of data to read
             user_space (bool): True if mem_addr is in user space
         """
-        # End address depending on the user_space flag
-        end_address = EdgePiMemoryInfo.USER_SPACE_END_BYTE.value if user_space \
-                      else (EdgePiMemoryInfo.USER_SPACE_START_BYTE.value - 1)
+        mem_total_size = (EdgePiMemoryInfo.USER_SPACE_END_BYTE.value+1) if user_space \
+                      else (EdgePiMemoryInfo.USER_SPACE_START_BYTE.value)
```

Comment:
97a9ec241e7eae9d3d21191a6fe48cc80ac44a09

good point, i created PRIVATE memory enums so it will calculate the max mem size of both user and private space. This way when the eeprom is replaced it can be handled by changing enum values